src/metabase/sync/interface.clj

```diff
@@ -51,3 +51,52 @@
 (def DatabaseInstance "Schema for a valid instance of a Metabase Database." (class Database))
 (def TableInstance    "Schema for a valid instance of a Metabase Table."    (class Table))
 (def FieldInstance    "Schema for a valid instance of a Metabase Field."    (class Field))
+
+
+;;; +------------------------------------------------------------------------------------------------------------------------+
+;;; |                                                SAMPLING & FINGERPRINTS                                                 |
+;;; +------------------------------------------------------------------------------------------------------------------------+
+
+(def ValuesSample
+  "Schema for a sample of VALUES returned by the `sample` sub-stage of analysis and passed into the `fingerprint` stage.
+   Guaranteed to be non-empty and non-nil."
+  ;; Validating against this is actually pretty quick, in the order of microseconds even for a 10,000 value sequence
+  (s/constrained [(s/pred (complement nil?))] seq "Non-empty sequence of non-nil values."))
+
+
+(def GlobalFingerprint
+  "Fingerprint values that Fields of all types should have."
+  {(s/optional-key :distinct-count) s/Int})
+
+(def Percent
```

Comment:
Not sure what the best name is, but "percent" makes me think 0-100%. Technically this might be called a "fraction", but that seems overloaded as well. 

@senior or @sb , do you guys have any alternative suggestions?